hyperledger-student-management.yaml

```
@@ -0,0 +1,232 @@
+openapi: "3.0.0"
+info:
+  description: "This is the Student API for UC4's hyperledger component."
+  version: "0.3.3"
```

Should be version 0.4.0, as we are in sprint 5.
```suggestion
  version: "0.4.0"
```